rainbowstream/draw.py

```diff
@@ -1066,21 +1066,27 @@ def format_quote(tweet):
     Quoting format
     """
     # Retrieve info
-    screen_name = '@' + tweet['user']['screen_name']
-    text = tweet['text']
+    screen_name = str( tweet['user']['screen_name'] )
+    text        = str( tweet['text'] )
+    tid         = str( tweet['id'] )
+
```

Comment:
`str` function is not compatible with Python 2 unicode type, in case `text` is non-English character (ex: Japanese).